urlbox-screenshots.php

```diff
@@ -480,11 +479,14 @@ public function urlbox_shortcode($atts)
 			$format = $urlboxOptions['format'];
 			$downloadedUrl = $url;
 			// $this->download_image($url, $format);
-			$output .= "<div class='" . esc_html__($urlboxOptions['div_class']) . "'>";
+			$output .= "<figure class='wp-block-image";

```

Comment:
in future we should probably have an option to disable adding this `wp-block-image` class (in case it breaks some themes) and an option to change the default tag from`<figure>`? no big deal for now though, seem like sensible defaults

Reply:
In theory, it shouldn't (WordPress chooses defaults very carefully in terms of backwards compatibility) - but obviously we can if we need ...